PINOperation.podspec

```diff
@@ -1,22 +1,25 @@
 Pod::Spec.new do |s|
   s.name          = 'PINOperation'
-  s.version       = '1.2.2'
+  s.version       = '1.2.3'
   s.homepage      = 'https://github.com/pinterest/PINOperation'
   s.summary       = 'Fast, concurrency-limited task queue for iOS and OS X.'
   s.authors       = { 'Garrett Moon' => 'garrett@pinterest.com' }
   s.source        = { :git => 'https://github.com/pinterest/PINOperation.git', :tag => "#{s.version}" }
   s.license       = { :type => 'Apache 2.0', :file => 'LICENSE.txt' }
   s.requires_arc  = true
   s.frameworks    = 'Foundation'
-  s.ios.deployment_target = '8.0'
-  s.osx.deployment_target = '10.8'
-  s.tvos.deployment_target = '9.0'
-  s.watchos.deployment_target = '2.0'
+  s.cocoapods_version = '>= 1.13.0'
```

Comment:
This is a common pattern used across other popular open source libraries, ensuring the VisionOS platform is recognized and Resource Bundles are properly generated.